velox/dwio/common/tests/TypeTests.cpp

```diff
@@ -43,20 +58,33 @@ TEST(TestType, selectedType) {
   EXPECT_EQ(0, typeWithId->id());
   EXPECT_EQ(11, typeWithId->maxId());
 
+  auto copySelector = [](size_t index) { return true; };
+  auto typeWithIdCopy = buildSelectedType(typeWithId, copySelector);
+  assertEqualTypeWithId(typeWithId, typeWithIdCopy);
+
   std::vector<bool> selected(12);
   selected[0] = true;
   selected[2] = true;
   auto selector = [&selected](size_t index) { return selected[index]; };
   auto cutType = buildSelectedType(typeWithId, selector);
+  assertEqualTypeWithId(typeWithIdCopy, typeWithId);
```

Comment:
Let's move this line to be after existing tests at line 77. Then call the assertValidTypeWithId() after this.